### ISSUE TITLE
Dlls have no version information.

### ISSUE THREAD

**Issue by jrbudda** [ValveSoftware/openvr#132]
Would it be possible to please add version information to the compiled dills, both here and installed with steam? I have several openvr_api.dll's floating around and no way to distinguish them.

Thanks.

**Comment by TheWhiteAmbit** [comment]
I guess most of the DLLs are not even different, but I would like to read this too.

**Comment by JoeLudwig** [comment]
openvr_api.dll will have a version number in it in the next SDK.

This DLL comes with the SDK, not with SteamVR itself and generally ships with the app. So the version number on the DLL says nothing about version of the runtime that will be loaded when the app calls VR_Init.

**Comment by Kyrluckechuck** [comment]
Is this satisfied enough to be closed?

**Comment by jrbudda** [comment]
No, they gave up updating the dll version 2 years ago. The dlls in the latest release still show as v 1.0.10.0, from 2017.
